APIAssignment1.py

```diff
@@ -0,0 +1,101 @@
+import json
+import requests
+import csv
+
+# 1. Write a function that would provide the below mentioned rocket and payload details
+# for the latest historical event.
+# Rocket name
+# Rocket height and diameter in meters
+# Payload id
+# Payload nationality and mass in kg
+def rocketdetails():
```

Comment:
function name is preferred to start with a verb in present tense, also try to think through on what verb to use, usually `get` is the last resort if there's no better option.
ex. `get` vs `fetch` vs `extract` etc

please also refer to this doc https://www.python.org/dev/peps/pep-0008/, search for `function name` section